package.json

```diff
@@ -211,8 +206,8 @@
     "final-form-calculate": "^1.3.2",
     "history": "4.10.1",
     "immortal-db": "^1.1.0",
-    "immutable": "^4.0.0-rc.12",
-    "js-cookie": "^2.2.1",
+    "immutable": "4.0.0-rc.12",
```

Comment:
Why removing the ^? Breaking changes after this version?

Reply:
They added weird types, for example, the return type for `toJS` is `Record<string, unknown` without a way to overwrite it. The benefits are small and long-term I think we want to get rid of this library at all. It's not even maintained anymore.